src/_includes/partials/rss-feed.njk

```diff
@@ -0,0 +1,9 @@
+<div class="rss-feed">
+  <h3>
+    <a href="/feed.xml">
+      <img src="/images/rss.svg">
+      RSS feed
+    </a>
```

Comment:
I'd like to see this vertical alignment cleaned up a bit.  In my browser I'd applied these styles to this a tag.
```css
a {
  display: flex;
  gap: 0.25rem;
  align-items: end;
}
```
Alter as needed 👍

Reply:
Thanks for the suggestion🙂 

## Before
![Screenshot from 2024-03-21 01-08-45](https://github.com/OpenWebAdvocacy/website/assets/1425259/c7ffda25-bbf4-4783-b3b1-a59db215930b)

## After
![Screenshot from 2024-03-21 01-08-49](https://github.com/OpenWebAdvocacy/website/assets/1425259/86af0d8f-a2b9-44ac-9d86-b9b69815b57e)

This looks much better!

Reply:
Could we write this as `Subscribe to the latest news from OWA`? I think it reads a little better to have `to` in there